### PR TITLE
PT-6965: Fix index building

### PR DESCRIPTION
**Pull request by einlied753**
## Description
Fix index building
### Jira-link:

 https://virtocommerce.atlassian.net/browse/PT-6965

### CODE REVIEW

File: src/VirtoCommerce.SearchModule.Web/Scripts/blades/indexes-list.js [hunk 1]
```diff
@@ -66,6 +66,20 @@ angular.module('virtoCommerce.searchModule')
...
+                                blade.currentEntities[i].canSwap = true;
+                                blade.currentEntities[i + 1].canSwap = true;
+                            }
+                            else {
+                                blade.currentEntities[i].canSwap = false;
+                                blade.currentEntities[i + 1].canSwap = false;
+                            }
+                        }
+                    }
+
```
Comment (ksavosteev) [inline src/VirtoCommerce.SearchModule.Web/Scripts/blades/indexes-list.js:82]: rewrite with grouping by documentType

**Review by ksavosteev — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.SearchModule.Web/Scripts/blades/indexes-list.tpl.html [hunk 2]
```diff
@@ -90,5 +87,4 @@
             </div>
         </div>
     </div>
-</script>
-
+</script>
```
Comment (Vectorfield4) [inline src/VirtoCommerce.SearchModule.Web/Scripts/blades/indexes-list.tpl.html:90]:
```suggestion
</script>

```

**Review by Vectorfield4 — APPROVED** [review]

**Review by getmansky — APPROVED** [review]

**Review by ksavosteev — COMMENTED** [review]
We still have the ability to call index via checkbox + Build Index button.

File: src/VirtoCommerce.SearchModule.Web/Scripts/blades/indexes-list.js [hunk 3]
```diff
@@ -110,6 +110,10 @@ angular.module('virtoCommerce.searchModule')
 
             // ui-grid
             $scope.setGridOptions = function (gridId, gridOptions) {
+                gridOptions.isRowSelectable = function (row) {
+                    return row.entity.isActive;
+                }
```
Comment (pushnitsa) [inline src/VirtoCommerce.SearchModule.Web/Scripts/blades/indexes-list.js:115]:
```suggestion
                gridOptions.isRowSelectable = (row) => row.entity.isActive;
```

**Review by ksavosteev — APPROVED** [review]

**Pull request merged**